### DISCUSSION TITLE
can't get the frontmatter inform via the string source, but can get the frontmatter when read from the 'mdx' files

### DISCUSSION THREAD

**Discussion by BertramYe** [hashicorp/next-mdx-remote#476] [Q&A]
dose anyone have meet the problem like here?
[can't get the frontmatter inform via the string source, but can get the frontmatter when read from the 'mdx' files](https://github.com/hashicorp/next-mdx-remote/issues/475)

**Comment by talatkuyuk** [comment]
Haven't you seen the answer [here](https://github.com/hashicorp/next-mdx-remote/issues/475#issuecomment-2395416037) ?

If it is okey, you can close the issue.
